### PR TITLE
Add Jenkinsfile for WinCI job (build only)

### PR DESCRIPTION
**Pull request by krdln**

### CODE REVIEW

File: CIScripts/Build/BuildFunctions.ps1 [hunk 1]
```diff
@@ -79,31 +90,42 @@ function Invoke-DockerDriverBuild {
...
            [Parameter(Mandatory = $true)] [string] $CertPath,
            [Parameter(Mandatory = $true)] [string] $CertPasswordFilePath,
-           [Parameter(Mandatory = $true)] [string] $OutputPath)
+           [Parameter(Mandatory = $true)] [string] $OutputPath,
+           [Parameter(Mandatory = $true)] [string] $LogsPath)
 
     $Job.PushStep("Docker driver build")
-    $Env:GOPATH=pwd
-    $srcPath = "$Env:GOPATH/src/$DriverSrcPath"
+    $GoPath = $Env:GOPATH
```
Comment (krdln) [inline CIScripts/Build/BuildFunctions.ps1:97]: What's the reason for moving setting `GoPath` to Jenkinsfile?

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,84 @@
+stage('Preparation') {
+    node('builder') {
+        deleteDir()
+        git branch: 'jfbuild', url: 'https://github.com/codilime/contrail-windows-tools/'
```
Comment (krdln) [inline Jenkinsfile]: Unfortunatelly, I've hardcoded the branch name here. The correct way would be to inherit the branch from where the `Jenkinsfile` is checked out.
Anyway, I think we should merge as is and fix it in PR #110.

**Comment by m-kostrzewa** [comment]
To reviewers: please verify local CI logs (because this job might've disable a build step or something after all)

**Review by sodar — DISMISSED** [review]

File: CIScripts/Test/TestRunner.ps1 [hunk 3]
```diff
@@ -19,6 +19,10 @@
 function Run-TestScenarios {
     Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
 
+    if(-not $Sessions) {
+        return
```
Comment (m-kostrzewa) [inline CIScripts/Test/TestRunner.ps1]: I think a warning should be printed here.

**Review by m-kostrzewa — DISMISSED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (wurbanski) [reply Jenkinsfile]: Is Jenkinsfile configured to be taken from gerrit/gh branch right now?

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        stash name: "WinArt", includes: "output/**/*"
+        stash name: "buildLogs", includes: "logs/**"
+    }
+}
+
+def SpawnedTestbedVMNames = ''
+
+stage('Provision') {
+    node('ansible') {
+        sh 'echo "Tu będzie ansible"'
```
Comment (wurbanski) [inline Jenkinsfile]: English? :D

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        // powershell script: './CIScripts/Test.ps1'
+    }
+}
+
+stage('Post-build') {
+    node('master') {
+        // cleanWs()
+        sh 'echo "Tu będzie cleanup środowiska"'
+        // unstash "buildLogs"
+        sh "echo rsync logs/ logs.opencontrail.org:${JOB_NAME}/${BUILD_ID}"
```
Comment (wurbanski) [inline Jenkinsfile]: Remove `echo` from this command to actually publish logs as it is required and use proper switches - this was just a placeholder.

**Review by wurbanski — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 5 above)
Comment (krdln) [reply Jenkinsfile]: Rsyncing (and capturing) actual logs will be done in the next pull request (probably in gerrit integration #110), so for now, let it stay as is.

File: CIScripts/Test/TestRunner.ps1 (hunk 3 above)
Comment (krdln) [reply CIScripts/Test/TestRunner.ps1]: I'm going to remove this early return (we abandoned the `map _ null = []` idea)

**Comment by m-kostrzewa** [comment]
if tests pass, LGTM

**Review by m-kostrzewa — APPROVED** [review]

**Review by sodar — APPROVED** [review]

**Review by wurbanski — APPROVED** [review]
LGTM

**Pull request merged**